src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,172 @@
+                        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+                        LocalDateTime parsedFromDateTime = LocalDateTime.parse(from, formatter);
+                        LocalDateTime parsedToDateTime = LocalDateTime.parse(to, formatter);
+                        Task newTask = new Event(content, parsedFromDateTime, parsedToDateTime);
+                        tasks.add(newTask);
+                        ui.add(newTask, tasks);
+                    } else if (splitedFromDateTime.length == 2 && splitedToDateTime.length == 1) {
+                        DateTimeFormatter formatter1 = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+                        DateTimeFormatter formatter2 = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+                        LocalDateTime parsedFromDateTime = LocalDateTime.parse(from, formatter1);
+                        LocalDate parsedToDate = LocalDate.parse(to, formatter2);
+                        Task newTask = new Event(content, parsedFromDateTime, parsedToDate);
+                        tasks.add(newTask);
+                        ui.add(newTask, tasks);
+                    } else if (splitedFromDateTime.length == 1 && splitedToDateTime.length == 2) {
+                        DateTimeFormatter formatter1 = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+                        DateTimeFormatter formatter2 = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+                        LocalDate parsedFromDate = LocalDate.parse(from, formatter2);
+                        LocalDateTime parsedToDateTime = LocalDateTime.parse(to, formatter1);
+                        Task newTask = new Event(content, parsedFromDate, parsedToDateTime);
+                        tasks.add(newTask);
+                        ui.add(newTask, tasks);
+                    } else {
+                        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+                        LocalDate parsedFromDate = LocalDate.parse(from, formatter);
+                        LocalDate parsedToDate = LocalDate.parse(to, formatter);
+                        Task newTask = new Event(content, parsedFromDate, parsedToDate);
+                        tasks.add(newTask);
+                        ui.add(newTask, tasks);
+                    }
+                } catch (Exception e) {
+                    throw new DukeException("Syntax of event: deadline {task description} /from ({yyyy-MM-dd HH:mm} or {yyyy-MM-dd}) "
+                            + "/to ({yyyy-MM-dd HH:mm} or {yyyy-MM-dd})\n"
+                            + "E.g. event exam /from 2022-12-31 15:00 /to 17:00");
+                }
+                break;
+            }
+        }
+        storage.saveChanges(tasks);
+    }
+
+    /**
+     * Returns False.
+
+     * @return False.
+     */
+    @Override
+    public boolean isExit() {
+        return false;
+    }
```

Comment:
if the superclass method already has a javadoc comment, no need to write comment for overridden method

Reply:
Oww alright, I saw from somewhere (can't recall) that we can use some tag, something like @inherit..., but I didn't manage to find it in my IDE. Any clue about it?

Reply:
I dont think such an annotation exists? You just don't have to write any comment at all for overridden methods